src/nyc_trees/apps/users/models.py

```diff
@@ -15,45 +15,124 @@
 class AchievementDefinition(object):
     """A class representing an achievment definition
     name - string - title of the achievement
-    description - string - description of the achievement requirements
+    description - requirements (after achievement)
+    description_future - requirements (before achievement)
     badge - path for a static image file
-    progress - a function which takes a user and returns a number from 0-100
+    achieved - a function which takes a user and returns a boolean
     """
-    def __init__(self, name, description, badge, progress):
+    def __init__(self, name, description, description_future, badge, achieved):
         assert isinstance(name, basestring)
         assert isinstance(description, basestring)
+        assert isinstance(description_future, basestring)
         assert isinstance(badge, basestring)
-        assert hasattr(progress, '__call__')
+        assert hasattr(achieved, '__call__')
 
         self.name = name
         self.description = description
+        self.description_future = description_future
         self.badge = badge
-        self.progress = progress
+        self.achieved = achieved
 
-    FINISH_TRAINING = 0
-    JOIN_10_GROUPS = 1
+    FOLLOW_GROUPS = 0
```

Comment:
We present `groups to follow` after online training is complete, should these be reversed?